gitlab/mixins.py

```diff
@@ -287,7 +278,7 @@ def create(
         if data is None:
             data = {}
 
-        self._check_missing_create_attrs(data)
+        _validate_attrs(data=data, attributes=self._create_attrs, excludes=None)
```

Comment:
Since this is in `utils.py` it needs to be imported for python to find it. Luckily, we already have `from gitlab import utils` at the top of the module so we can just do this

```suggestion
        utils._validate_attrs(data=data, attributes=self._create_attrs)
```
(Also, since we default to `None` for excludes, we can leave it out).

Reply:
applied.